### ISSUE TITLE
Converter is dropping final letter of cell if directly adjacent to next cell delimiter

### ISSUE THREAD

**Issue by mojavelinux** [opendevise/downdoc#11]
If a table cells ends with a letter, and that letter is directly adjacent to the next cell delimiter, that letter is being dropped during conversion.

Consider the following case:

```
|===
|foo|bar|baz
|===
```

This results in:

```
|     |     |     |
| --- | --- | --- |
| fo | ba | ba |
```

We expect the result to be:

```
|     |     |     |
| --- | --- | --- |
| foo | bar | baz |
```

What's happening is that the last letter is being consumed as the cell style. The cell style must be preceded by a space or newline.

**Comment by mojavelinux** [comment]
Resolved by d06cddd035fa7b17ea23fd244faa78fcf9e06496

**Comment by mojavelinux** [comment]
Related to this issue, the converter should consume the alignment modifiers that precede style cell style (e.g., `>s`). Note, however, that these settings are not honored. We just don't want them to be treated as content.
